src/Raven.Studio/typescript/models/database/index/indexDefinition.ts

```diff
@@ -398,7 +407,7 @@ class indexDefinition {
         }
     }
     
-    static empty(): indexDefinition {
+    static empty(indexingDatabaseSettings?: Record<indexingDatabaseSettings[number], models.serverWideOnlyEntry | models.databaseEntry<string | number>>): indexDefinition {
```

Comment:
If it's 'empty' then we'd rather not pass any argument and just use an empty object instead of indexingDatabaseSettings